.travis.yml

```diff
@@ -126,16 +126,18 @@ script:
     fi
   - |
     if [[ "${DOCUMENTATION}" == "true" ]]; then
-      cd ..
-      doxygen doxy/doxygen.config
-      # Suppress output to avoid leaking the token when the command fails
-      git clone https://arximboldi:${GH_TOKEN}@github.com/ableton/atria --depth 1 --branch=gh-pages gh-pages &>/dev/null
-      rm -rf gh-pages/{search,*.png,*.css,*.js,*.html}
-      cp -R output-doc/* gh-pages/
-      pushd gh-pages
-      git add --all .
-      git commit --allow-empty -m "Update to ${TRAVIS_COMMIT:0:7}"
-      # Suppress output to avoid leaking the token
-      travis_retry git push origin gh-pages &>/dev/null
-      popd
+      if [[ "${TRAVIS_PULL_REQUEST}" == "false" && "${TRAVIS_BRANCH}" == "master" ]]; then
+        cd ..
+        doxygen doxy/doxygen.config
+        # Suppress output to avoid leaking the token when the command fails
+        git clone https://arximboldi:${GH_TOKEN}@github.com/ableton/atria --depth 1 --branch=gh-pages gh-pages &>/dev/null
```

Comment:
The clone is the official clone, but done with my user account because I generated the token with it. I thought that might me less risky and bureaucracyful than generating the token from an official Ableton account. I guess I could try the worktree thing and change the remote, admittedly I just followed what the referred links do...